src/chat/participant.ts

```diff
@@ -169,18 +172,20 @@ export async function handleChatMessage(
   while (continueConversation && iterations < maxIterations) {
     continueConversation = false;
 
+    // NOTE: uncomment for local debugging
+    // debugLogChatMessages(messages);
```

Comment:
May be handy to set this up behind a setting or `when` clause, but it can easily take up the entire output channel window if left enabled